ui/src/app/pages/workspace/workspace-about.spec.tsx

```diff
@@ -299,4 +299,19 @@ describe('WorkspaceAbout', () => {
         .props().disabled
     ).toBeTruthy();
   });
+
+  it('should display Google project id', async () => {
+    const wrapper = component();
+    await waitOneTickAndUpdate(wrapper);
+    expect(wrapper.find('[data-test-id="googleProject"]').text()).toContain(
+      workspace.googleProject
+    );
+  });
+  it('should display Google project id', async () => {
```

Comment:
copy-paste: should be bucket name

Reply:
good catch.: https://github.com/all-of-us/workbench/pull/8154